default.yml

```diff
@@ -41,11 +41,5 @@ Metrics/BlockLength:
 Metrics/LineLength:
   Max: 120
 
-Metrics/AbcSize:
-  Max: 18
-
-Style/ClassAndModuleChildren:
```

Comment:
move discussion to the separate issue https://github.com/datarockets/datarockets-style/issues/19